hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/client/DefaultCertificateClient.java

```diff
@@ -647,7 +650,8 @@ protected enum InitCase {
     PRIVATE_KEY,
     PRIVATEKEY_CERT,
     PUBLICKEY_PRIVATEKEY,
-    ALL
+    ALL,
+    EXPIRES
```

Comment:
Please change the documentation of this method to match the current state.

Reply:
I would suggest use EXPIRED_CERT instead of "EXPIRES" here.

Reply:
Done.

Reply:
The documentation still does not mention the EXPIRED_CERT case, and refers to this ENUM as it has only 8 elements. If we refactor this in [HDDS-7376](https://issues.apache.org/jira/browse/HDDS-7376) then we can proceed with this, but I'd prefer if the documentation was consistent at all times.

Reply:
I see, I have updated the enum's doc as well.